### PR TITLE
Update Ceridian Dayforce connector reference page

### PR DESCRIPTION
**Pull request by gabilang**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ comprehensive human capital management system that covers the entire employee li
 benefits, talent management, workforce management, and services. The entire system resides on cloud that takes the 
 burden of managing and replicating data on-premise.
 
-## What you'll build
+## What You'll Build
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:8]: Use sentence style for titles.  https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 2]
```diff
@@ -19,89 +19,230 @@ Both of the two operations are exposed via an API. The API with the context `/da
 * `/postEmployeeContact`  - This will create the contact information of an employee when invoked. The relevant 
 parameters must be passed in the body as we will see below.
 
-## Setting up the environment 
+## Setting Up the Environment 
 
 Please follow the steps mentioned at [Setting up Ceridian Dayforce Environment]({{base_path}}/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-config/) document in order to create a Ceridian Dayforce developer account and obtain credentials you need to access the 
 Dayforce sample APIs. Keep them saved to be used in the next steps.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:25]:
```suggestion
Dayforce sample APIs.Keep them saved for use in the next steps.  

```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 3]
```diff
@@ -19,89 +19,230 @@ Both of the two operations are exposed via an API. The API with the context `/da
...
+    |---------------------|-----------------------------------|
+    | username            | json-eval($.username)             |
+    | password           | json-eval($.password)             |
+    | clientNamespace    | json-eval($.clientNamespace)      |
+    | apiVersion         | json-eval($.apiVersion)           |
+    | contextDateRangeFrom | json-eval($.contextDateRangeFrom) |
+    | contextDateRangeTo | json-eval($.contextDateRangeTo)   |
+    | xRefCode           | json-eval($.xRefCode)             |
+
+    You can add these properties by clicking on **Add Parameter** of the **Property Group** tab. When entering the `Property Value` fields, make sure to select the `Ex` button to specify the value types as expressions.
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:73]:
```suggestion
    You can add these properties by clicking on the **Add Parameter** of the **Property Group** tab. When entering the `Property Value` fields, make sure to select the `Ex` button to specify the value types as expressions.
```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 4]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
...
-    </resource>
-</api>
-```
+Dayforce sample APIs. Keep them saved for use in the next steps.  
+
+## Setting up the integration project
+
+Follow the steps in [create integration project]({{base_path}}/develop/create-integration-project/) to set up the Integration Project.
+
+## Creating the integration logic
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:31]:
```suggestion
## Create the integration logic
```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 5]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
...
-         <send/>
-      </inSequence>
-        <outSequence/>
-        <faultSequence/>
-    </resource>
-</api>
-```
+Dayforce sample APIs. Keep them saved for use in the next steps.  
+
+## Setting up the integration project
```
Comment (anupama-pathirage) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:27]:
```suggestion
## Set up the integration project
```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 6]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
 ## Setting up the environment
```
Comment (chanikag) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:22]:
```suggestion
## Set up the environment 
```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 7]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
...
+    | Parameter name | Parameter value (as expression) |
+    |----------------|---------------------------------|
+    | xRefCode       | $ctx:xRefCode                   |
+    | contextDate    | $ctx:contextDate                |
+    | contextDateRangeFrom | $ctx:contextDateRangeFrom   |
+    | contextDateRangeTo | $ctx:contextDateRangeTo       |
+
+    Click on **Submit** to save the configurations.
+
+10. Add a **Send** mediator to the mediation logic to send the response back to the client.
```
Comment (chanikag) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:109]:
```suggestion
10. Add a **Respond** mediator to the mediation logic to send the response back to the client.
```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 8]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
...
+				    <password>{$ctx:password}</password>
+				    <clientNamespace>{$ctx:clientNamespace}</clientNamespace>
+				    <apiVersion>{$ctx:apiVersion}</apiVersion>
+			    </ceridiandayforce.init>
+			    <ceridiandayforce.postEmployeeContacts>
+				    <xRefCode>{$ctx:xRefCode}</xRefCode>
+				    <isValidateOnly>{$ctx:isValidateOnly}</isValidateOnly>
+				    <fieldAndValue>{$ctx:fieldAndValue}</fieldAndValue>
+			    </ceridiandayforce.postEmployeeContacts>
+			    <send/>
```
Comment (chanikag) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:236]: Use respond mediator

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 9]
```diff
@@ -113,20 +254,9 @@ You can download the ZIP file and extract the contents to get the project code.
...
-You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-micli).
-
-??? note "Click here for instructions on deploying on WSO2 Enterprise Integrator 6"
-    1. You can copy the composite application to the `<PRODUCT-HOME>/repository/deployment/server/carbonapps` folder and start the server.
-
-    2. WSO2 EI server starts and you can login to the Management Console https://localhost:9443/carbon/ URL. Provide login credentials. The default credentials will be admin/admin. 
+In order to deploy and run the project, refer the [build and run]({{base_path}}/develop/deploy-artifacts/#build-and-run) guide.  
 
-    3. You can see that the API is deployed under the API section. 
+You can further refer the application deployed through the CLI tool. See the instructions on [managing integrations from the CLI]({{base_path}}/observe-and-manage/managing-integrations-with-apictl).
```
Comment (chanikag) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:259]: This page is renamed to micli. Please check

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 10]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
...
+			    <send/>
+		    </inSequence>
+		    <faultSequence>
+		    </faultSequence>
+	    </resource>
+    </api>
+    ```
+
+## Exporting integration logic as a CApp
+In order to export the project, refer to the [build and export the carbon application]({{base_path}}/develop/deploy-artifacts/#build-and-export-the-carbon-application) guide.
```
Comment (chanikag) [inline en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:245]:
```suggestion
In order to export the project, refer to the [build and export the composite application]({{base_path}}/develop/deploy-artifacts/#build-and-export-the-carbon-application) guide.
```

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md (hunk 7 above)
Comment (chanikag) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:109]: You can point to the Respond mediator docs in this space here.

[Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/)

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md [hunk 11]
```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
...
+
+    | Parameter name | Parameter value (as expression) |
+    |----------------|---------------------------------|
+    | xRefCode       | $ctx:xRefCode                   |
+    | isValidateOnly | $ctx:isValidateOnly             |
+    | fieldAndValue  | $ctx:fieldAndValue              |
+
+    Click on **Submit** to save the configurations.
+
+19. Add a **Send** mediator to the mediation logic to send the response back to the client.
```
[the remark this answers is not in the document]
Comment (chanikag) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:177]: You can point to the Respond mediator docs in this space here.

[Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/)

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md (hunk 6 above)
Comment (gabilang) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:22]: In the guideline, it has been suggested to use the `-ing` form in the titles.

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md (hunk 5 above)
Comment (gabilang) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:27]: In the guideline, it has been suggested to use the `-ing` form.

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md (hunk 9 above)
Comment (gabilang) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md]: Updated accordingly.

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md (hunk 6 above)
Comment (gabilang) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md:22]: The guideline has been revised to use simple present tense

File: en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md (hunk 5 above)
Comment (gabilang) [reply en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md]: The guideline has been revised to use the simple present tense.

**Review by chanikag — APPROVED** [review]

**Pull request merged**